DataGenerator/src/main/java/org/opensearch/migrations/data/workloads/HttpLogs.java

```diff
@@ -83,8 +83,9 @@ public Stream<ObjectNode> createDocs(int numDocs) {
         return IntStream.range(0, numDocs)
             .mapToObj(i -> {
                 var random = new Random(i);
+                long randomTime = randomTime(currentTime, random);
                 return mapper.createObjectNode()
-                    .put("@timestamp", randomTime(currentTime, random))
+                    .put("@timestamp", randomTime)
```

Comment:
Seeing similar fixes throughout this change, what was the issue here, too much complexity?

Reply:
> Remove this use of "put"; it is deprecated.

This one is silly - boo on Jackson for deprecating an overridden method, and boo on SonarQube for not identifying the correct override without a specific typed local.